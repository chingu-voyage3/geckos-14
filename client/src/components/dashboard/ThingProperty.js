import React from 'react';

const ThingProperty = props => {
  // console.log('Device Component');
  return (
    <div className="thing-property">
      <div>{props.display}</div>
      <div className="thing-description">{props.description}</div>
    </div>
  );
};

export default ThingProperty;
