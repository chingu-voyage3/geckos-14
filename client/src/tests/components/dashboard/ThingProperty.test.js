import React from 'react';
import ThingProperty from '../../../components/dashboard/ThingProperty';
import { shallow } from 'enzyme';
// import * as d from '../../../tempData';

it('renders without crashing', () => {
  const wrapper = shallow(<ThingProperty />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
