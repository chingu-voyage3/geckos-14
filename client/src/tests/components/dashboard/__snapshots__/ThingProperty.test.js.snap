// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="thing-property"
>
  <div />
  <div
    className="thing-description"
  />
</div>
`;
